### DISCUSSION TITLE
YOLOv8 Training output : "loss: 0.0000e+00 - box_loss: 0.0000e+00 - class_loss: 0.0000e+00"

### DISCUSSION THREAD

**Discussion by MagiPrince** [keras-team/keras-cv#1879] [Q&A]
Hi everyone,

I am trying to work with YOLOv8 and train it on a custom dataset that I have. I ran the example code just fine, but when I am lowering the size of my images and the width, height of the objects to detect, the training outputs : `loss: 0.0000e+00 - box_loss: 0.0000e+00 - class_loss: 0.0000e+00`.
I'm working with images of size 64x64 where I try to detect some element that are 8 pixels large in width and height.

I used the parameter "center_xywh" for my bounding boxes, where I give the x, y center with the w, h in pixels.

It seems that the training works when the size of the w, h is bigger than a given number of pixels, but I don't get it all...

There is an example of data that produces the same weird behaviour : 

```
images = np.ones((2, 64, 64, 3)) # or np.array(tf.ones(shape=(2, 128, 128, 3)))
labels = {
    "boxes": np.array([
        [
            [15, 14, 10, 10],
            [20, 39, 10, 10],
            [17, 29, 10, 10],
        ],

        [
            [25, 43, 10, 10],
            [23, 32, 10, 10],
            [32, 32, 10, 10],
        ]
    ]),
    "classes": np.array([[1, 1, 1], [1, 1, 0]]),
}
```

May someone help me with that ?

Thanks !

**Comment by jbischof** [comment]
Thanks for reaching out @MagiPrince! I'm not sure I understand your question. What is the expected behavior for your data? Could you file an issue with an example colab so we can debug?

**Reply by MagiPrince** [reply]
Hi @jbischof, basically my question is : is it normal that the loss is equal to `0.0000e+00` when I train the model ? I was expecting, since I train from scratch, to have a big loss first that decrease over the iterations.

If you tell me that this shouldn't append I'll file an issue !

**Reply by jbischof** [reply]
This does sound odd! Please file an issue with a minimal colab attached replicating the unexpected behavior. Thanks!

**Comment by MagiPrince** [comment]
For those that are facing the same issue, I solved my problem by editing the value of `BOX_REGRESSION_CHANNELS` constant to 16 in the `yolo_v8_detector.py` file.
